common/src/main/java/tachyon/Constants.java

```diff
@@ -39,6 +39,14 @@
   public static final String MESOS_RESOURCE_MEM = "mem";
   public static final String MESOS_RESOURCE_DISK = "disk";
   public static final String MESOS_RESOURCE_PORTS = "ports";
+  public static final String MESOS_EXECUTOR_DEPENDENCY_PATH
```

Comment:
by tachyon convention, this String const should be named after `INTEGRATION_MESOS_EXECUTOR_DEPENDENCY_PATH`, so for other strings definition below.